readthedocs/builds/version_slug.py

```diff
@@ -78,13 +81,37 @@ def get_queryset(self, model_cls, slug_field):
                 return model._default_manager.all()
         return model_cls._default_manager.all()
 
+    def _normalize(self, content):
+        """
+        Normalize some invalid characters to become a dash (``-``).
```

Comment:
Could use a better docstring -- this tells me _what_ not _why. Is there a reason we only do this for a subset of characters, but not others?

Reply:
I added the _why_: which is basically to keep compatibility.